modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -1328,7 +1328,9 @@ function getIFramePageOutput( ){
 
 <script type="text/javascript">
     var customCSS = <?php echo $customCss ?>;
-    if (['kWidget'] && !window['kWidget'].isMobileDevice() && customCSS){
+    var skin =  window["kalturaIframePackageData"].playerConfig.layout ? window["kalturaIframePackageData"].playerConfig.layout.skin : "kdark";
+    var mobileSkin = window['kWidget'].isChromeCast() || ( window["kalturaIframePackageData"].playerConfig.vars["EmbedPlayer.EnableMobileSkin"] === true && skin === "kdark" && window['kWidget'].isMobileDevice() && !window['kWidget'].isWindowsPhone() );
```

Comment:
U are accessing nested object so need to check if exist:
* window["kalturaIframePackageData"].playerConfig.layout => check that both window["kalturaIframePackageData"], window["kalturaIframePackageData"].playerConfig, window["kalturaIframePackageData"].playerConfig.vars exist
* window['kWidget'].* => check that window['kWidget'] exist
* Anything else I missed.